node/blockAPI/baseBlock.go

```diff
@@ -126,3 +137,17 @@ func (bap *baseAPIBockProcessor) getFromStorerWithEpoch(unit dataRetriever.UnitT
 	storer := bap.store.GetStorer(unit)
 	return storer.GetFromEpoch(key, epoch)
 }
+
+func (bap *baseAPIBockProcessor) getBlockStatus(storerUnit dataRetriever.UnitType, blockAPI *blockStr.APIBlock) (string, error) {
+	nonceToByteSlice := bap.uint64ByteSliceConverter.ToByteSlice(blockAPI.Nonce)
```

Comment:
duplicated code with statusComputer.

Reply:
for the moment I don't have a solution for this.